text/0000-re-org-docs.md

```diff
@@ -0,0 +1,265 @@
+# Meta
+[meta]: #meta
+- Name: Reorganize docs website
+- Start Date: (fill in today's date: 2023-09-20)
+- Author(s): natalieparellano
+- Status: Draft <!-- Acceptable values: Draft, Approved, On Hold, Superseded -->
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: N/A
+
+# Summary
+[summary]: #summary
+
+This RFC proposes reorganizing the docs website according to CNB personas and the "four quadrants of docs" aka the [Divio documentation system](https://documentation.divio.com/structure.html).
+
+# Definitions
+[definitions]: #definitions
+
+* Personas - people with different roles who interact with the CNB project in different ways
+  * Buildpack Authors - write buildpacks
+  * App Developers - use buildpacks
+  * Platform Operators - orchestrate buildpacks builds
+  * ...
+* Four quadrants
+  * Tutorials - from Divio: `lessons that take the reader by the hand through a series of steps to complete a project of some kind`
+    * Oriented toward beginners
+    * Should be tested to make sure they work
+  * Explanations - from Divio: `discussion [that] clarify and illuminate a particular topic`
+    * Theoretical knowledge
+  * How-to guides - from Divio: `take the reader through the steps required to solve a real-world problem`
+    * Practical knowledge
+  * Reference - from Divio: `technical descriptions of the machinery and how to operate it` (similar to CNB spec)
+    * Technical knowledge
+
+# Motivation
+[motivation]: #motivation
+
+- Why should we do this?
+  - Our docs website is a bit disorganized - it's hard to find stuff, and it's hard to add stuff. Consequently, the content of our docs is woefully incomplete.
+- What is the expected outcome?
+  - Our docs website will be easier to use for everyone. Eventually, the content of our docs will become more complete as it's easier to add stuff.
+
+# What it is
+[what-it-is]: #what-it-is
+
+This provides a high level overview of the feature.
+
+- Define the target persona: all personas, including project contributors, can benefit from this.
+- Explaining the feature largely in terms of examples.
+  - See the example [branch](https://github.com/buildpacks/docs/tree/refactor) of github.com/buildpacks/docs and abbreviated tree with notes below.
+
+# How it Works
+[how-it-works]: #how-it-works
+
+```
+.
+├── concepts <- NOTE: this corresponds to the explanation quadrant; this content would be largely unchanged (just reorganized slightly)
```

Comment:
This tree is very deep.  I suspect we need a secondary top-level menu with your, Concepts, Buildpacks for App Developers, Buildpacks Authors and Platform Operators entries.  That way each sub-menu displays his own sub-tree?

Reply:
Updated in adb31d5